ublox-short-range/src/wifi/supplicant.rs

```diff
@@ -291,4 +365,8 @@ where
             .map(WifiConnection::is_connected)
             .unwrap_or_default()
     }
+
+    pub fn flush(&mut self) -> Result<(), WifiConnectionError> {
+        todo!()
```

Comment:
Is this intended?

Reply:
The intention is a function removing all stored credentials, as it is not being used the writing of it has been postphoned to https://github.com/BlackbirdHQ/ublox-short-range-rs/pull/51.